ocs_ci/ocs/jenkins.py

```diff
@@ -296,11 +296,13 @@ def create_ocs_jenkins_template(self):
         tmp_dict['labels']['app'] = 'jenkins-persistent-ocs'
         tmp_dict['labels']['template'] = 'jenkins-persistent-ocs-template'
         tmp_dict['metadata']['name'] = 'jenkins-persistent-ocs'
-        tmp_dict['objects'][1]['metadata']['annotations'] = {
-            'volume.beta.kubernetes.io/storage-class': 'ocs-storagecluster-ceph-rbd'
-        }
-        tmp_dict['objects'][2]['spec']['template']['spec']['containers'][0]['env'].append(
-            {'name': 'JAVA_OPTS', 'value': '${JAVA_OPTS}'})
+        # Find pvc in objects list
+        for i in range(len(tmp_dict['objects'])):
```

Comment:
I assume you are looping here because Kind: 'PersistentVolumeClaim' position in the objects list differs in OCP 4.5 and OCP 4.6.
Please add a comment above mentioning it for reference.